app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+        if person.get("husband"):
+            Person.people[person["name"]].husband \
+                = Person.people[person["husband"]]
```

Comment:
try to avoid using linebreakers here, u can break the line with some brackets
please, use a code style as mentioned in this file:
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md